test/integration/targets/wait_for/tasks/main.yml

```diff
@@ -62,13 +62,36 @@
     search_regex: completed
     timeout: 20
   register: waitfor
-- name: verify test wait for port timeout
+
+- name: verify test wait for keyword in file
   assert:
     that:
-      - waitfor is successful
-      - "waitfor.search_regex == 'completed'"
-      - waitfor.elapsed >= 5
-      - waitfor.elapsed <= 15
+    - waitfor is successful
+    - "waitfor.search_regex == 'completed'"
+    - waitfor.elapsed >= 5
+    - waitfor.elapsed <= 15
```

Comment:
The original indentation on these 4 lines is OK and doesn't need to be changed.

Reply:
Reverted